src/components/BottomBar/BottomBar.js

```diff
@@ -42,9 +53,62 @@ export const BottomBar = () => {
 
   const selectedNote = useSelector(state => state.notesMindMap.selectedNote);
 
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+
+  const handleClick = event => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
```

Comment:
Let's move the code for menu to a separate file (separate component).
Right now it's confusing why does `BottomBar` have `open`, `anchorEl` and `setAnchorEl`, `handleClick` and `handleClose`.
If we keep this code in `BottomBar.js` it would be a bit more logical to name these variables `isMenuOpen`, `menuAnchorEl`, `setMenuAnchorEl`, `handleOpenMenuButtonClick`, `handleMenuClose`. Because it is all related to the `Menu`, not to `BottomBar` itself.

If we extract this code to a separate file: `BottomBarMenu.js`, then we can don't need to add the "menu" word to every variable name, and the code will be more readable and clean.

Reply:
Thank you for the good comment. Totally agree with you. I moved the menu-related code to the separated file